authors/apps/articles/tests/test_views.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.test import APIClient
+from rest_framework import status
+from django.test import TestCase
+from django.urls import reverse
+from authors.apps.articles.models import Article
+
+
+class CreateArticleTestCase(TestCase):
+    """This class defines the test suit for the Article model"""
+    pass
+
+
+class ViewArticleTestCase(TestCase):
+    """This class defines the test suit for api views"""
+    pass
```

Comment:
You had a previous PR with test on views. Maybe you should have this reflect in that PR instead

Reply:
Let me check on that.

Reply:
That seems to had been reflected on @kamyaD PR. We are working it.